lib/swagger.js

```diff
@@ -162,8 +162,6 @@ SwaggerApi.prototype.build = function() {
       }
     }
   };
-  var e = (typeof window !== 'undefined' ? window : exports);
-  e.authorizations.apply(obj);
```

Comment:
I believe that if you remove this line, the authorizations won't be applied when fetching an individual api declaration